saleor/order/tests/test_fetch_order_prices.py

```diff
@@ -1378,108 +1382,6 @@ def test_fetch_order_prices_voucher_specific_product_fixed(
     assert line_discount.value == discount_amount
 
 
-def test_fetch_order_prices_voucher_specific_product_discount_line_object_updated(
```

Comment:
This test was replaced with `test_add_manual_line_discount_order_with_voucher_specific_product` and `test_draft_order_update_with_voucher_specific_product_and_manual_line_discount`